ci/features/compile.feature

```diff
@@ -1,9 +0,0 @@
-Feature: Compile script
```

Comment:
Should we leave a test here that validates that this script exits for now? I could go either way.

Reply:
I'd say let's just leave it out. It's such a simple script that if anything goes wrong it'd probably be something not covered by the test we would right.